openmetadata-ui/src/main/resources/ui/src/utils/EntityUtils.tsx

```diff
@@ -550,5 +543,5 @@ export const isColumnTestSupported = (dataType: string) => {
 };
 
 export const getTitleCase = (text?: string) => {
-  return text ? startCase(camelCase(text)) : '';
+  return text ? startCase(text) : '';
```

Comment:
Why are we making changes to this?

Reply:
User messages or description texts should not have `camelCase` in it
CC: @devyani-kaushik